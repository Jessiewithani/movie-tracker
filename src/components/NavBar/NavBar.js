import './NavBar.scss';
import React from 'react';

const NavBar = () => {
  return (
    <nav className="nav-bar">
      <h1 className="main-logo">Movie<span className='logo-second'>Trac</span></h1>
      <div className="menu">
        <span className="menu-item">Favorite Movies</span>
        <span className="menu-item">Prob another thing</span>      
        <span className="menu-item">Login/Create Account</span>
      </div>  
    </nav>
  )
}

export default NavBar;