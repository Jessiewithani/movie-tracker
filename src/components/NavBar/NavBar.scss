@import url('https://fonts.googleapis.com/css?family=Josefin+Slab&display=swap');
@import url('https://fonts.googleapis.com/css?family=Cabin&display=swap');
@import url('https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600&display=swap');
@import url('https://fonts.googleapis.com/css?family=Nanum+Gothic:400,700|Nanum+Myeongjo:400,700&display=swap');

// font-family: 'Montserrat', sans-serif;
// font-family: 'Nanum Gothic', sans-serif;
// font-family: 'Nanum Myeongjo', serif;
// font-family: 'Liu Jian Mao Cao', cursive;

.nav-bar {
  background-color: black;
  color: lightgray;
  height: 12vh;
  font-size: 2em;
  vertical-align: middle;
}

.main-logo {
  // font-family: 'Josefin Slab', serif;
  font-family: 'Nanum Myeongjo', serif; //not bad
  // font-family: 'Nanum Gothic', sans-serif;
  // font-family: 'Montserrat', sans-serif;



  float: left;
  // display: inline-block;

  // .logo-second {
  //   font-family: 'Cabin', sans-serif;
  // }
}

.menu {
  display: flex;
  float: right;
  font-size: .8em;

  .menu-item {
    margin: 0 13px;
    margin-top: 28px;
    // font-family: 'Josefin Slab', serif;
    font-family: 'Nanum Myeongjo', serif;
    // font-family: 'Nanum Gothic', sans-serif; //too plain????
    // font-family: 'Montserrat', sans-serif;


    color: lightgray;
    font-weight: bold;
    text-decoration: none;
  }
}

.hidden-letters {
  display: inline;
}

.hidden-letters:after {
  width: 0px;
  overflow: hidden;
  display: inline-block;
  vertical-align: middle;
  transition: .4s linear;
}
.hidden-letters:nth-child(1):after {
  content: 'ovie ';
}
.hidden-letters:nth-child(2):after {
  content: 'racker ';
}
.hidden-div:hover .hidden-letters:nth-child(1):after {
  width: 90px;
}
.hidden-div:hover .hidden-letters:nth-child(2):after {
  width: 130px;
}

.hidden-div {
  margin-top: 28px;
  margin-left: 15px;
}

