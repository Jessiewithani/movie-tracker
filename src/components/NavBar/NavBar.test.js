import React from 'react';
import NavBar from './NavBar';
import { shallow } from 'enzyme';

describe('NavBar', () => {
  it('should match a snapshot', () => {
    const wrapper = shallow(
      <NavBar />
    )
  expect(wrapper).toMatchSnapshot()  
  })
})