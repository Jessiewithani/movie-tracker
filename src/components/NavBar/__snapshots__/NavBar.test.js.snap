// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match a snapshot 1`] = `
<nav
  className="nav-bar"
>
  <h1
    className="main-logo"
  >
    <div
      className="hidden-div"
    >
      <span
        className="hidden-letters"
      >
        M
      </span>
      <span
        className="hidden-letters"
      >
        T
      </span>
    </div>
  </h1>
  <div
    className="menu"
  >
    <Link
      className="menu-item"
      to="/login"
    >
      Login
    </Link>
    <Link
      className="menu-item"
      to="/new-user"
    >
      Create New Account
    </Link>
    <Link
      className="menu-item"
      to="/favorites"
    >
      Favorite Movies
    </Link>
    <Link
      className="menu-item"
      to="/"
    >
      Sign Out
    </Link>
  </div>
</nav>
`;
